qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/commons/JavaParameterInfo.java

```diff
@@ -21,12 +21,16 @@
  */
 public class JavaParameterInfo extends JavaElementInfo {
 
+	private static final String VARARGS = "...";

```

Comment:
Same as above, since you use it twice, except this time on the ls side

Reply:
Qute LS has not QuteJavaConstants. More this constants is used only in JavaParameterInfo, so w e don't need to expose it in an Java constants file as public.